templates/patterns/text_featured_media/pattern-text-featured-media.html.twig

```diff
@@ -37,37 +37,41 @@
 {% endif %}
 
 {% if image or video or text or call_to_action_link or text_title %}
-  {% if title %}
-    <h2 class="ecl-u-type-heading-2 ecl-u-mt-2xl ecl-u-mt-m-3xl ecl-u-mb-l">{{ title }}</h2>
-  {% endif %}
-  <div class="ecl-row">
-    {% if image or video %}
-      <div class="ecl-col-m-6 ecl-u-mb-m ecl-u-mb-m-none{{ media_position_class }}">
-        {% if image %}
-          {% include '@ecl-twig/media-container' with {
-            'image': image.src,
-            'alt': image.alt,
-            'description': caption,
-          } %}
-        {% elseif video %}
-          {% include '@ecl-twig/media-container' with {
-            'embedded_media': video,
-            'ratio': video_ratio,
-            'description': caption,
-          } %}
-        {% endif %}
-      </div>
-    {% endif %}
-    {% if text_title or text or call_to_action_link %}
-      <div class="{{ image or video ? 'ecl-col-m-6' : 'ecl-col-12' }}">
-        {% if text_title %}
-          <div class="ecl-u-type-prolonged-m ecl-u-type-bold ecl-u-type-color-grey">{{ text_title }}</div>
-        {% endif %}
-        {% if text %}
-          <div class="ecl">{{ text }}</div>
-        {% endif %}
-        {{ call_to_action_link }}
-      </div>
+  <div class="ecl-featured-item{{ highlighted == true ? ' ecl-featured-item--extended' : '' }}">
+    {% if title %}
+      <div class="ecl-featured-item__heading">{{ title }}</div>
     {% endif %}
+    <article class="ecl-featured-item__container">
+      {% if image or video %}
+        <div class="ecl-col-m-6 ecl-u-mb-m ecl-u-mb-m-none{{ media_position_class }}">
+          {% if image %}
+            {% include '@ecl-twig/media-container' with {
+              'image': image.src,
+              'alt': image.alt,
+              'description': caption,
+            } %}
+          {% elseif video %}
+            {% include '@ecl-twig/media-container' with {
+              'embedded_media': video,
+              'ratio': video_ratio,
+              'description': caption,
+            } %}
+          {% endif %}
+        </div>
+      {% endif %}
+      {% if text_title or text or call_to_action_link %}
+        <div class="ecl-featured-item__item">
+          <div class="{{ image or video ? 'ecl-col-m-6' : 'ecl-col-12' }}">
+            {% if text_title %}
+              <div class="ecl-featured-item__title">{{ text_title }}</div>
+            {% endif %}
+            {% if text %}
+              <div class="ecl">{{ text }}</div>
```

Comment:
```suggestion
              <div class="ecl-featured-item__description ecl">{{ text }}</div>
```

Reply:
We can't use this class, check the FRONT ticket linked in our ticket.

Reply:
If you are talking about https://citnet.tech.ec.europa.eu/CITnet/jira/browse/FRONT-3561, could you please check PR https://github.com/ec-europa/europa-component-library/pull/2351/files and playground https://ecl-preview-2351--europa-component-library.netlify.app/playground/ec/?path=/story/components-featured-item--extended.

Class `ecl-featured-item__description` is still used. Changes are just in the removing of font size.

Reply:
> If you are talking about https://citnet.tech.ec.europa.eu/CITnet/jira/browse/FRONT-3561, could you please check PR https://github.com/ec-europa/europa-component-library/pull/2351/files and playground https://ecl-preview-2351--europa-component-library.netlify.app/playground/ec/?path=/story/components-featured-item--extended.
> 
> Class `ecl-featured-item__description` is still used. Changes are just in the removing of font size.

I see it's fixed but not released, what should we do? Block the ticket on the ECL upgrade? ECL still needs to do improvements for this component so we can drop all our customisation and use their component.

The other comment is out of scope, indeed. The links are properly working and the scope of the ticket was to integrate the highlighted display which has been done.

Reply:
Do we need to create a ticket just for one class? Can't we simply remove it while we rebuild whole template?

Reply:
We can't use ECL component since it doesn't support left/right image and links instead of button.